tests/test_views.py

```diff
@@ -13,6 +11,11 @@ def create_articles(amount):
         PublicArticleFactory()
 
 
+def publish_articles_with_publisher(articles):
+    for article in articles:
+        article.publish()
```

Comment:
There was a ready management command in django-model-publisher which eventually looped through objects and called publish method one by one. Would it make sense if there was a custom queryset for example which could handle mass publishing? For example `Article.publisher_manager.drafts().publish()` and this would loop through the objects and punlish them one by one.